spec/controllers/teams_controller_spec.rb

```diff
@@ -142,15 +144,31 @@
     it "editing of a team description" do
       old_description = team.description
       expect do
-        patch :update, id: team.id, team: { description: "new description" }, format: "js"
+        patch :update, id: team.id, team: { name:        team.name,
+                                            description: "new description" }, format: "js"
       end.to change(PublicActivity::Activity, :count).by(1)
 
       team_description_activity = PublicActivity::Activity.find_by(
         key: "team.change_team_description")
       expect(team_description_activity.owner).to eq(owner)
       expect(team_description_activity.trackable).to eq(team)
-      expect(team_description_activity.parameters[:old_description]).to eq(old_description)
-      expect(team_description_activity.parameters[:new_description]).to eq("new description")
+      expect(team_description_activity.parameters[:old]).to eq(old_description)
+      expect(team_description_activity.parameters[:new]).to eq("new description")
+    end
+
+    it "editing of the team name" do
+      old_name = team.name
+      expect do
+        patch :update, id: team.id, team: { name:        "new name",
```

Comment:
This looks kind of weird. What I'd do:

```ruby
team = { name: "new name", description: team.description }
patch :update, id: team.id, team: team, format: "js"
```

Yes, they're two lines too, but they are more readable I think.